docs/components/component_props.md

```diff
@@ -1,7 +1,7 @@
 ```python exec
```

Comment:
Can this page be called `props.md` or is that a name conflict?

Reply:
renamed